### PR TITLE
Adding toasts

### PR DESCRIPTION
**Pull request by painTojn1**

### CODE REVIEW

**Comment by DLowHP** [comment]
Jest crunchips jest impreza

File: src/components/DeleteBook.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const DeleteBook: FC<IBookProps> = ({ id, onBookDelete, title }) => {
       .from('books')
       .delete()
       .match({ id });
-      setErr(error);
+    setErr(error);
```
Comment (DLowHP) [inline src/components/DeleteBook.tsx:19]: Is it necessary to store it in useState? You could just execute toast there
```suggestion
    error && errorToast(msg, id);
```

**Review by DLowHP — CHANGES_REQUESTED** [review]
a słyszałeś kiedyś krzyk pterodaktyla?

**Review by wps0 — APPROVED** [review]

File: src/views/BookListViewAdminStore.tsx [hunk 2]
```diff
@@ -28,6 +29,13 @@ const BookListViewAdminStore = () => {
     };
     fetchData();
   }, [filter]);
+
+  useEffect(() => {
+    if (error) {
```
Comment (wps0) [inline src/views/BookListViewAdminStore.tsx:34]: I'd say that useEffect is unnecessary here and it's better to just fire the toast immediately after getting an error.

**Review by wps0 — CHANGES_REQUESTED** [review]

**Review by DLowHP — APPROVED** [review]
It's okay now

**Review by wps0 — APPROVED** [review]

**Pull request merged**